faststream/rabbit/response.py

```diff
@@ -1,5 +1,22 @@
+from typing import TYPE_CHECKING, Optional
+
 from faststream.broker.response import Response
 
+if TYPE_CHECKING:
+    from faststream.rabbit import RabbitQueue
+    from faststream.rabbit.types import AioPikaSendableMessage
+
 
 class RabbitResponse(Response):
-    pass
+    def __init__(
+        self,
+        body: "AioPikaSendableMessage",
+        *,
+        correlation_id: Optional[str] = None,
+        queue: Optional["RabbitQueue"] = None,
```

Comment:
Remove `queue`, `routing_key` and RPC options. Add `headers`

Reply:
Also, you forgot about RMQ extra options: there are a lot of them - https://github.com/Flosckow/faststream/blob/feat-extend-response-api/faststream/rabbit/publisher/usecase.py#L220

Reply:
Yes, indeed, I missed that. We are talking about Publish Kwargs, which also need to be passed to method init?

Reply:
Yep